### ISSUE TITLE
Add code to repo

### ISSUE THREAD

**Issue by carsin** [elijaharita/dotsgame#11]
@javaseruntimeenvironment
help

**Comment by elijaharita** [comment]
yo kids

**Comment by elijaharita** [comment]
whatre you doing here

**Comment by elijaharita** [comment]
this was for the old node server which im porting to c++ right now

join the new one http://76.88.185.140:13049/

**Comment by elijaharita** [comment]
oh wait i killed it hold on

**Comment by elijaharita** [comment]
FIXED

**Comment by carsin** [comment]
its blocked on school wifi xd imma open a new issue for that

**Comment by carsin** [comment]
#12 
maybe @javaseruntimeenvironment or @dabit778 can help
